docs/faq/adding-your-project.md

```diff
@@ -0,0 +1,12 @@
+# How to add your project to CodeMC?
+
+CodeMC is open for join requests, as we want to offer a free and reliable CI-Service to everyone that needs one.
+
+In order to join will you need to join the [Discord Server](https://discord.gg/AGcFMu6). There, head to the `#request-access` channel and read the Pinned Message as it contains important info on what requirements you need to fulfill and how to request being added.
+
+/// note
+You are only required to make one join request with a project.  
+If you are accepted to the CI, you can create new projects by yourself without additional approval.
+
+With that said, abusing this freedom will have severe punishments as a result.
```

Comment:
Maybe be specific about what they are?

Reply:
I'll probs add a mention of the guidelines and its punishments section.

Reply:
Improved the note.